distro/src/main/java/io/camunda/migrator/app/MigratorApp.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.migrator.app;
+
+import io.camunda.migrator.HistoryMigrator;
+import io.camunda.migrator.RuntimeMigrator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.DefaultApplicationArguments;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ConfigurableApplicationContext;
+
+@SpringBootApplication
+public class MigratorApp {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(MigratorApp.class);
+
+  public static void main(String[] args) {
+    ConfigurableApplicationContext context = SpringApplication.run(MigratorApp.class, args);
+    ApplicationArguments appArgs = new DefaultApplicationArguments(args);
+    try {
+      if (appArgs.containsOption("runtime")) {
```

Comment:
/nitpick We could move this and "history" constants into `protected static final` fields.

Reply:
Good point, updated. https://github.com/camunda/c7-data-migrator/pull/15/commits/4a6e4cc634620313aa53eea8b2c76f10433f154a